packages/echo-base/src/utils/storage.ts

```diff
@@ -3,11 +3,7 @@ import { parseJsonWithDate } from './jsonParseUtils';
 
 export const storage: EchoLocalStorage = {
     setItem: <T>(key: string, data: T) => {
-        if (typeof data === 'string') {
-            localStorage.setItem(key, data);
-        } else {
-            localStorage.setItem(key, JSON.stringify(data));
-        }
+        localStorage.setItem(key, JSON.stringify(data));
```

Comment:
is this backward compatible with existing values that we save and get?

Reply:
All values that are not string have the same behaviour. The only reason it was working for strings in the past, was that this try/catch block was throwing an error since you can not JSON.parse() a normal string, and we returned the data as it is (a string, since it was fetched from localstorage). Now, this block doesn't throw an error, but it correctly parses the stringifyed string into a normal string, leaving us with the same result.
![image](https://user-images.githubusercontent.com/74668394/155993702-e19bd863-88a0-4f87-a990-66eb1ef0f6dd.png)